src/mapgen/treegen.cpp

```diff
@@ -292,29 +292,29 @@ treegen::error make_ltree(MMVManip &vmanip, v3s16 p0,
 
 	Key for Special L-System Symbols used in Axioms
 
-    G  - move forward one unit with the pen up
-    F  - move forward one unit with the pen down drawing trunks and branches
-    f  - move forward one unit with the pen down drawing leaves (100% chance)
-    T  - move forward one unit with the pen down drawing trunks only
-    R  - move forward one unit with the pen down placing fruit
-    A  - replace with rules set A
-    B  - replace with rules set B
-    C  - replace with rules set C
-    D  - replace with rules set D
-    a  - replace with rules set A, chance 90%
-    b  - replace with rules set B, chance 80%
-    c  - replace with rules set C, chance 70%
-    d  - replace with rules set D, chance 60%
-    +  - yaw the turtle right by angle degrees
-    -  - yaw the turtle left by angle degrees
-    &  - pitch the turtle down by angle degrees
-    ^  - pitch the turtle up by angle degrees
-    /  - roll the turtle to the right by angle degrees
-    *  - roll the turtle to the left by angle degrees
-    [  - save in stack current state info
-    ]  - recover from stack state info
-
-    */
+	  G  - move forward one unit with the pen up
+	  F  - move forward one unit with the pen down drawing trunks and branches
+	  f  - move forward one unit with the pen down drawing leaves (100% chance)
+	  T  - move forward one unit with the pen down drawing trunks only
+	  R  - move forward one unit with the pen down placing fruit
+	  A  - replace with rules set A
+	  B  - replace with rules set B
+	  C  - replace with rules set C
+	  D  - replace with rules set D
+	  a  - replace with rules set A, chance 90%
+	  b  - replace with rules set B, chance 80%
+	  c  - replace with rules set C, chance 70%
+	  d  - replace with rules set D, chance 60%
+	  +  - yaw the turtle right by angle degrees
+	  -  - yaw the turtle left by angle degrees
+	  &  - pitch the turtle down by angle degrees
+	  ^  - pitch the turtle up by angle degrees
+	  /  - roll the turtle to the right by angle degrees
+	  *  - roll the turtle to the left by angle degrees
+	  [  - save in stack current state info
+	  ]  - recover from stack state info
+
+	 */
```

Comment:
I don't see any benefits of this change.

Reply:
This was a situation when the beginning of a multiline comment was indented by one tabulator, and the rest of it was indented by spaces.
The check process is sensitive to this now. So it was reported as an error.

If it is a problem, I can ignore multiple comments and ignore their indents.

Reply:
But disabling it means that indents like here [`src/util/string.h`](https://github.com/minetest/minetest/pull/15002/files/aa19202aad794b9c6ee9520d7cb75c504eb18da5#diff-71badc1cc71ba46244f7841a088251bb294265f4fe9e662c0ad6b15be540eee4) and  here [`src/unittest/test_map_settings_manager.cpp`](https://github.com/minetest/minetest/pull/15002/files/aa19202aad794b9c6ee9520d7cb75c504eb18da5#diff-3fd3459a4043212e05ed20dc3563b4f114f4e48b9d7b58dcbc84b0bf36efc83a) will not be recognized as errors.

Reply:
Hm, I'm fine with both, but I guess it would make sense to allow spaces here, like for the big comment in src/server/clientiface.h

Reply:
In `clientiface.h` it is not a problem because the first `/*` is not indented.

So, I will keep it for now, if somebody else disagrees with this solution, I change it here and in the script.

Reply:
Imo new version makes sense.